src/components/misc/globalToolbar.tsx

```diff
@@ -1,18 +1,44 @@
 import React from 'react'
+import { Link } from 'react-router-dom'
 
 import DarkModeToggle from 'components/utils/darkModeToggle'
+import FaIcon from 'components/shared/icons/faIcon'
 import UserOptionsDropdown from 'components/utils/userOptionsDropdown'
 
+import { useAppSelector } from 'redux/hooks'
+
 import styles from './globalToolbar.scss'
 
-const GlobalToolbar = () => (
-  <div className={styles.bar}>
-    <h1>Auto Four</h1>
-    <div className={styles.controls}>
-      <UserOptionsDropdown />
-      <DarkModeToggle />
+const GlobalToolbar = () => {
+  const userId = useAppSelector((store) => store.user.id)
+
+  return (
+    <div className={styles.bar}>
+      <h1>Auto Four</h1>
```

Comment:
should link the big h1 or future logo to the home page

Reply:
Good call, I forgot to do this.